Src/Modules/MarkdownPS/Public/New-MDAlert.Tests.ps1

```diff
@@ -24,7 +24,7 @@ Describe -Tag @('MarkdownPS', 'Cmdlet', 'Public', 'New-MDAlert') 'New-MDAlert' {
         @('Line 1') | New-MDAlert | Should -Be $expected
     }
     It '-Lines count is 2 & -Style not specified' {
-        $expected = '> [!NOTE]' + $newLine + '> Line 1' + $newLine + '> Line 2' + $newLine + $newLine
+        $expected = '> [!NOTE]' + $newLine + '> Line 1' + $newLine + '>' + $newLine + '> Line 2' + $newLine + $newLine
         # New-MDAlert -Lines @('Line 1', 'Line 2') | Should -Be $expected
```

Comment:
This is a left over comment from initial implementation

Reply:
Got it, uncommented, tests still passing fine.